recipes/unionml/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "unionml" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unionml-{{ version }}.tar.gz
+  sha256: e2d906402dcf3710f0fbc6726e764d5cd21569a1f23864ae2fcbf97198119125
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - unionml = unionml.cli:app
+  script:
+    ## Ensure inclusion of requirements.txt
```

Comment:
Is this really needed?

Reply:
I pushed a PR in upstream to include the requirements.txt and it was merged. So, I would request you to merge it as is for now. Once the next release of unionml is available on PyPI, I will update the recipe and remove the workaround.

- unionml PR: https://github.com/unionai-oss/unionml/pull/125

Reply:
I didn't know this was being written and did my own recipe; you shouldn't need this script here. My recipe worked without it.

Reply:
CF: https://github.com/conda-forge/staged-recipes/pull/19414

Reply:
@thewchan Thank you, for the suggestion. Removed the workaround.